syntaxhighlighter2/scripts/shBrushCpp.js

```diff
@@ -54,7 +54,8 @@ SyntaxHighlighter.brushes.Cpp = function()
 					'jmp_buf mbstate_t _off_t _onexit_t _PNH ptrdiff_t _purecall_handler ' +
 					'sig_atomic_t size_t _stat __stat64 _stati64 terminate_function ' +
 					'time_t __time64_t _timeb __timeb64 tm uintptr_t _utimbuf ' +
-					'va_list wchar_t wctrans_t wctype_t wint_t signed';
+					'va_list wchar_t wctrans_t wctype_t wint_t signed' +

```

Comment:
The word `signed` needs a whitespace character after it, otherwise this is broken for the keywords `signed` and `int8_t`.

Reply:
I added the missing whitespace. Sorry about that.